packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -6,27 +6,27 @@ import 'dart:math';
 
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
 
-/// A widget that takes two children, lays them out along [axis], and allows
-/// the user to resize them.
+/// A widget that takes a list of children, lays them out along [axis], and
+/// allows the user to resize them.
 ///
 /// The user can customize the amount of space allocated to each child by
 /// dragging a divider between them.
 ///
-/// [initialFirstFraction] defines how much space to give the [firstChild]
-/// when first building this widget. [secondChild] will take the remaining
-/// space.
+/// [initialFractions] defines how much space to give each child when building
+/// this widget.
 ///
-/// The user can drag the widget with key [dividerKey] to change
-/// the space allocated between [firstChild] and [secondChild].
-// TODO(djshuckerow): introduce support for a minimum fraction a child is allowed.
+/// The user can drag the widget with key [dividerKey[index]] to change the
```

Comment:
nit: make this comment more generic. Just indicate the splitters can be dragged rather than explaining the details of the splitter in the class level comment.

Reply:
done